backend/models/user.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema({
+  id: {
+    type: Number,
+    required: true,
+    unique: true,
+    index: true,
+    primary: true
+  },
+  password: {
+    type: String,
+    required: true
+  },
+  email: {
```

Comment:
add a lowercase option to make sure email addresses aren't case sensitive

```suggestion
  email: {
  lowercase: true,
```